src/main/java/gary/Gary.java

```diff
@@ -0,0 +1,183 @@
+package gary;
+
+import gary.exception.*;
+import gary.task.Deadline;
+import gary.task.Event;
+import gary.task.Task;
+import gary.task.Todo;
+
+import java.util.Scanner;
+
+public class Gary {
+    public static final int MAX_TASK = 100;
+
+    public static final int TODO_DESCRIPTION_START_INDEX = 5;
+    public static final int DEADLINE_DESCRIPTION_START_INDEX = 9;
+    public static final int DEADLINE_BY_SPACE_LENGTH = 4;
+    public static final int EVENT_DESCRIPTION_START_INDEX = 6;
+    public static final int EVENT_FROM_SPACE_LENGTH = 6;
+    public static final int EVENT_TO_SPACE_LENGTH = 4;
+    public static void main(String[] args) {
```

Comment:
You could consider using consistent spacing (betwen variables, between methods, etc.) throughout the code